src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
+
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import { useState } from 'react';
+import Header from './components/Header';
+import HomePage from './components/HomePage';
+import ArticlesTray from './components/ArticlesTray';
+import ArticlePage from './components/ArticlePage';
+import UsersPage from './components/Users/UsersPage';
+import Footer from './components/Footer';
+
 
 function App() {
+  const [loggedInUser, setLoggedInUser] = useState('');
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header setLoggedInUser={setLoggedInUser} loggedInUser={loggedInUser} />
+      <Routes>
```

Comment:
Awesome choice of paths for these endpoints, they follow a great pattern for a user to follow